src/main/java/org/haffson/adventofcode/days/day01/MyFileNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package org.haffson.adventofcode.days.day01;
+
+public class MyFileNotFoundException extends RuntimeException {
```

Comment:
There is room for improvement regarding the name :-)

E.g. `UncheckedFileNotFoundException` as I guess that's your intention here.